### ISSUE TITLE
Using Class weights while training on Imbalanced training data

### ISSUE THREAD

**Issue by Prak11** [experiencor/keras-yolo2#405]
I have imbalanced annotations for my dataset. I want to introduce class weights in the fit_generator for every batch that I get after  the train_generator is defined in train.py file. How do I get the information on classes/labels in every batch and the count which I can give class_weights to the fit_generator ?
